api/src/main/java/org/openmrs/module/metadatadeploy/bundle/CoreConstructors.java

```diff
@@ -56,6 +60,16 @@
 public class CoreConstructors {
 
 	/**
+	 * Constructs a dbsessionFactory source
+	 * @return the transient object
+	 */
+	public static DbSessionFactory dbSessionFactory(SessionFactory sessionFactory) {
```

Comment:
@dkayiwa the directions given in the documentation instruct us to include any Missing object in the  CoreConstructors class.I tried browsing through the Data Access Layer but i was surprised it was absent.